modules/landing_zone/scripts/apply.py

```diff
@@ -51,16 +46,13 @@ def extractOutputValues(result):
 def getOutputValueByType(value):
     if isinstance(value, string_types):
         return value
-    else:
+    elif isinstance(value, list):
         return ','.join(map(str, value))
-
-def execWithErrors(args_list):
-    for args in args_list:
-        p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=os.environ['root'])
-        (result, error) = p.communicate()
-        if p.wait() != 0:
-            print("Error: failed to execute command:")
-            raise Exception(error)
+    else:
+        response = []
+        for (key, val) in value.items():
+            response.append(key+'='+getOutputValueByType(val))

```

Comment:
I think spacing would look much better
```suggestion
            response.append(key + '=' + getOutputValueByType(val))
```